org.eclipse.scout.rt.ui.html.app/src/main/java/org/eclipse/scout/rt/ui/html/app/UiServletContributors.java

```diff
@@ -80,6 +78,15 @@ public void contribute(ServletContextHandler handler) {
     }
   }
 
+  @Order(4000)
+  public static class MultipartConfigFilterContributor implements IServletFilterContributor {
```

Comment:
UiServletMultipartConfigFilterContributor
@Order(100)
JavaDoc stating that usually multipart config is registered by servletHolder.getRegistration().setMultipartConfig